src/reference/asciidoc/whats-new.adoc

```diff
@@ -115,9 +115,16 @@ See <<http-inbound>> for more information.
 
 ==== SFTP Changes
 
+===== Factory Bean
 A new factory bean is provided to simplify the configuration of Jsch proxies for SFTP.
 See <<sftp-proxy-factory-bean>> for more information.
 
+===== chmod
+
+The SFTP outbound gateway (for `put` and `mput` commands) and the SFTP outbound channel adapter now support the
+`chmod` attribute to change the remote file permissions after uploading.
+See <<sftp-outbound-channel-adapter>> and <<sftp-outbound-gateway>> for more information.
```

Comment:
There is no such a chapter.
Should be `<<sftp-outbound>>` according to the `sftp.adoc`.